redbot/core/commands/help.py

```diff
@@ -540,25 +548,24 @@ def width_maker(cmds):
         coms = {c async for c in self.help_filter_func(ctx, ctx.bot.walk_commands())}
         fuzzy_commands = await fuzzy_command_search(ctx, help_for, commands=coms, min_score=75)
         use_embeds = await ctx.embed_requested()
+        
         if fuzzy_commands:
-            ret = await format_fuzzy_results(ctx, fuzzy_commands, embed=use_embeds)
-            if use_embeds:
-                ret.set_author(name=f"{ctx.me.display_name} Help Menu", icon_url=ctx.me.avatar_url)
-                tagline = (await ctx.bot._config.help.tagline()) or self.get_default_tagline(ctx)
-                ret.set_footer(text=tagline)
-                await ctx.send(embed=ret)
-            else:
-                await ctx.send(ret)
+            msg = await format_fuzzy_results(ctx, fuzzy_commands, embed=use_embeds)
```

Comment:
If `use_embeds=True`, `format_fuzzy_results` will return `discord.Embed` so this doesn't work properly with L560 where you create a new embed using `msg` var as embed description.

Reply:
Yep.... Will handle consolidating more of the logic another time...